internal/pkg/describe/env_test.go

```diff
@@ -38,43 +43,47 @@ func TestEnvDescriber_FilterAppsForEnv(t *testing.T) {
 	}
 	testApp1 := &archer.Application{
 		Project: "testProject",
-		Name: "testApp1",
-		Type: "load-balanced",
+		Name:    "testApp1",
+		Type:    "load-balanced",
 	}
-	testApp3 :=	&archer.Application{
+	testApp3 := &archer.Application{
 		Project: "testProject",
 		Name:    "testApp3",
-		Type: "load-balanced",
+		Type:    "load-balanced",
 	}
 	env1apps := []*archer.Application{testApp1, testApp3}
 	badTags := map[string]string{stack.EnvTagKey: "badEnvName"}
+	goodTags := map[string]string{stack.EnvTagKey: "testEnv1"}
+	mockError := errors.New("some error")
 	testCases := map[string]struct {
-		env *archer.Environment
-		proj *archer.Project
-		apps []*archer.Application
+		//env  *archer.Environment
+		//proj *archer.Project
+		//apps []*archer.Application
 
-		setupMocks func(mocks envDescriberMocks)
-		mockrgClient	func(m *mocks.MockresourceGroupsClient)
+		setupMocks   func(mocks envDescriberMocks)
+		mockrgClient func(m *mocks.MockresourceGroupsClient)
 
-		wantedApps   []*archer.Application
+		wantedApps  []*archer.Application
 		wantedError error
 	}{
 		"return error if fail to get ARN": {
```

Comment:
nit: perhaps more descriptive to say something like "errors if cannot retrieve resource ARNS by tags" ?